### PR TITLE
Update mandatory-user-profile.md

### PR DESCRIPTION
**Pull request by warren-msft**
We need to also update screen shots with the control panel applet that has the check box.

### CODE REVIEW

File: windows/client-management/mandatory-user-profile.md [hunk 1]
```diff
@@ -84,6 +84,8 @@ First, you create a default user profile with the customizations that you want,
 
 1. Right-click Start, go to **Control Panel** (view by large or small icons) > **System** > **Advanced system settings**, and click **Settings** in the **User Profiles** section.
 
+Starting in Windows 10 version (2004) Open the Settings app and click on Advanced system settings
```
Comment (JohanFreelancer9) [inline windows/client-management/mandatory-user-profile.md:87]:
```suggestion
Starting in Windows 10, version 2004, open the **Settings** app and select **Advanced system settings**.
```

File: windows/client-management/mandatory-user-profile.md [hunk 2]
```diff
@@ -84,6 +84,8 @@ First, you create a default user profile with the customizations that you want,
 
 1. Right-click Start, go to **Control Panel** (view by large or small icons) > **System** > **Advanced system settings**, and click **Settings** in the **User Profiles** section.
```
Comment (JohanFreelancer9) [inline windows/client-management/mandatory-user-profile.md:85]:
```suggestion
1. Right-click Start, go to **Control Panel** (view by large or small icons) > **System** > **Advanced system settings**, and click **Settings** in the **User Profiles** section. Alternatively, starting in Windows 10, version 2004, open the **Settings** app and select **Advanced system settings**.
```

File: windows/client-management/mandatory-user-profile.md [hunk 3]
```diff
@@ -92,12 +94,13 @@ First, you create a default user profile with the customizations that you want,
...
+1. In **Select User or Group**, in the **Enter the object name to select** field, type `everyone` or the group of users that the profile will be assigned to, click **Check Names**, and then click **OK**.
 
 1. In **Copy To**, in the **Copy profile to** field, enter the path and folder name where you want to store the mandatory profile. The folder name must use the correct [extension](#profile-extension-for-each-windows-version) for the operating system version. For example, the folder name must end with ".v6" to identify it as a user profile folder for Windows 10, version 1607.
 
    - If the device is joined to the domain and you are signed in with an account that has permissions to write to a shared folder on the network, you can enter the shared folder path.
    - If the device is not joined to the domain, you can save the profile locally and then copy it to the shared folder location.
+   - Optionally you can check the box “Mandatory profile”. This step is not required but will set permissions that are more restrictive and it is recommend to do so
```
Comment (JohanFreelancer9) [inline windows/client-management/mandatory-user-profile.md:103]:
```suggestion
   - Optionally, you can check the **Mandatory profile** checkbox. This step is not required but will set permissions that are more restrictive and we recommend doing so.
```

File: windows/client-management/mandatory-user-profile.md (hunk 2 above)
Comment (JohanFreelancer9) [reply windows/client-management/mandatory-user-profile.md:85]: Suggestion mainly to keep the numbering of the steps intact.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by illfated** [comment]
This PR targets the file `windows/client-management/mandatory-user-profile.md` which is the same file used in PR #9139 — likely reason for the file conflict in the other PR.

**Comment by e0i** [comment]
Hello @warren-msft, The PR is good to go once the copy editing suggestions have been applied. Thank you for the contribution.

**Pull request merged**